compiler/z/codegen/OMRCodeGenerator.hpp

```diff
@@ -948,6 +948,8 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
 
    bool opCodeIsNoOpOnThisPlatform(TR::ILOpCode &opCode);
 
+   bool supportsPrefetch() { return getS390ProcessorInfo()->supportsArch(TR_S390ProcessorInfo::TR_z10); }
```

Comment:
Fixed in 79e4476. I removed the query altogether as there are currently no consumers of it. The `TR::Prefetch` IL evaluators on all platforms have architecture level set checks before emitting instructions for a prefetch which renders this query obsolete.